opm/common/data/SimulationDataContainer.cpp

```diff
@@ -75,6 +75,36 @@ namespace Opm {
     }
 
 
+    void SimulationDataContainer::setCellDataComponent( const std::string& key , size_t component , const std::vector<int>& cells , const std::vector<double>& values) {
```

Comment:
Formatting nitpicking: this line is too long, if-blocks should have braces and one should use c++ style comments.